src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageMetadataControl.xaml.cs

```diff
@@ -31,4 +34,38 @@ private void PackageMetadataControl_DataContextChanged(object sender, Dependency
             }
         }
     }
+
+    /// <summary>
+    /// DateTime Converter for published date in package metadata.
+    /// </summary>
+    public class DateTimeConverter : IValueConverter
+    {
+        /// <summary>
+        /// Converts DateTime object into custom formatting.
+        /// Goal of the converter is to have custom formatting for japanese since "D" formatting does not include day of the week.
+        /// </summary>
+        /// <returns>string value of the date time.</returns>
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (value == null)
+            {
+                return null;
+            }
+
+            var dateTime = DateTime.Parse(value.ToString());
+            if (string.Equals(culture.Name, "ja-JP"))
```

Comment:
is there no better way of knowing that the culture is Japanese?

Reply:
I dont think so. The [sample](https://msdn.microsoft.com/en-us/library/ky2chs3h(v=vs.110).aspx#Examples) for creating new cultures also use strings.